contracts/ERC20_AUTOLOAD_SIMPLE.sol

```diff
@@ -74,21 +73,31 @@ contract CypherAutoLoad is Pausable, AccessControl, ReentrancyGuard {
     function setMaxWithdrawalLimit(
         uint newLimit
     ) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        _maxWithdrawalLimit = newLimit;
+        _maxWithdrawalLimitPerTransaction = newLimit;
+        emit MaxWithdrawalLimit(_msgSender(), newLimit);
     }
 
     function getMaxWithdrawalLimit() external view returns (uint) {
-        return _maxWithdrawalLimit;
+        return _maxWithdrawalLimitPerTransaction;
     }
 
-    function pause() external whenNotPaused anyPrivilagedUser {
+    function pause() external whenNotPaused anyPrivilegedUser {
         _pause();
     }
 
     function unpause() external whenPaused onlyRole(DEFAULT_ADMIN_ROLE) {
         _unpause();
     }
 
+    function safeTransfer(
+        IERC20 token,
+        address userAddress,
+        address beneficiaryAddress,
+        uint amount
+    ) external {
+        token.safeTransferFrom(userAddress, beneficiaryAddress, amount);
```

Comment:
safeTransferFrom doesn't return any boolean value, hence cannot be used in require. Moved it to an external function to wrap it in try catch